src/lib/y2partitioner/dialogs/main.rb

```diff
@@ -104,6 +106,63 @@ def run(system, initial)
       def hostname
         @hostname ||= Yast::Hostname.CurrentHostname
       end
+
+      # Checks whether the dialog should be rendered again
+      #
+      # @return [Boolean]
+      def continue_running?(result)
+        if result == :redraw
+          true
+        elsif result == :abort && Yast::Mode.installation
+          !Yast::Popup.ConfirmAbort(:painless)
+        else
+          false
+        end
+      end
+
+      # Whether the user has accepted the Partitioner warning
+      #
+      # @note It returns true if the partitioner warning is not shown.
+      #
+      # @return [Boolean]
+      def accepted_partitioner_warning?
+        return true unless show_partitioner_warning?
+
+        partitioner_warning == :yes
+      end
+
+      # Popup to alert the user about the usage of the Partitioner
+      #
+      # @return [Symbol] user's answer (:yes, :no)
+      def partitioner_warning
+        # Warning popup about using the expert partitioner
+        message = _(
+          "This is for experts only.\n" \
+          "You might lose support if you use this!\n\n" \
```

Comment:
"You might lose support if you use this!" is at one time a bold claim (loosing support for a product) and also very unspecific.

How about something along the lines of "Not all partition setups will be supported."?

Reply:
Also, the openSUSE branch does not have support at all.

Reply:
Yes, cbf641f is bsc#1088960. I will indicate it in the changelog (I didn't know we had already a bug for it).

Reply:
Fine with me, but the wording is copied from old storage. Do you think we should change it now?

Reply:
Yes, I'm aware of it. It still strikes me as odd.

As said for the openSUSE part we don't provide support, so you can't really loose it. And for SLES the customer has a real contract. And I'm not sure we can just claim such a thing.

Just a thought. Treat it as NP, if you want.

Reply:
I was just reminded that this warning is intended specifically for caasp and there it is true.

So, please ignore this.

Reply:
Well, know it will also be displayed for Kubic, I guess. In that context, the usage of "product" and "lose support" becomes tricky. @sysrich Will kubic use `expert_partitioner_warning`?

Reply:
@ancorgs yes, Kubic will use that warning, primarily because it's also needed by CaaSP and Kubic is exploring new methods to be considered for the CaaSP installation

In both cases I'm fine with the wording because it just simply wont work if (for example) you setup Kubic or CaaSP without btrfs and snapshots and readonly rootfs :)